mods/d2k/chrome.yaml

```diff
@@ -100,6 +100,7 @@ flags: buttons.png
 	smuggler: 111,114,23,21
 	fremen: 133,114,23,21
 	worm: 156,114,23,21
+	mercenary: 179,114,20,21
```

Comment:
I just retested it. `22x21` is correct (at least for `atreides`). You're able to see a bit of the `harkonnen` flag using `23x21`.